.github/workflows/publish-commit-bottles.yml

```
@@ -95,17 +95,35 @@ jobs:
           pushable="$(jq .maintainer_can_modify <<< "$pr_data")"
           branch="$(jq --raw-output .head.ref <<< "$pr_data")"
           remote="$(jq --raw-output .head.repo.clone_url <<< "$pr_data")"
+          head_repo="$(jq --raw-output .head.repo.full_name <<< "$pr_data")"
+          head_repo_owner="$(jq --raw-output .head.repo.owner.login <<< "$pr_data")"
           fork_type="$(jq --raw-output .head.repo.owner.type <<< "$pr_data")"
 
-          if [ -z "$pushable" ] || [ -z "$branch" ] || [ -z "$remote" ] || [ -z "$fork_type" ]
+          if [ -z "$pushable" ] ||
+             [ -z "$branch" ] ||
+             [ -z "$remote" ] ||
+             [ -z "$head_repo" ] ||
+             [ -z "$head_repo_owner" ] ||
+             [ -z "$fork_type" ]
           then
             echo "::error ::Failed to get PR data!"
             exit 1
           fi
 
-          echo "branch=$branch" >> "$GITHUB_OUTPUT"
+          if [ "$branch" = "master" ]
+          then
+            echo "branch=$head_repo_owner/$branch" >> "$GITHUB_OUTPUT"
```

```suggestion
            echo "branch=$head_repo_owner/master" >> "$GITHUB_OUTPUT"
```
Could probably just propagate the constant here.